src/helpers/getSubCategoryOfHeroIndicator.js

```diff
@@ -1,13 +1,13 @@
 export default function getSubCategoryOfHeroIndicator(category) {
-  let subCategoryName = null;
+  let subCategory = null;
```

Comment:
Mentioned above, returning `null` should not be valid, should warn or throw an error in that case (assuming it's covered by tests)